velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -286,12 +288,23 @@ HiveConnectorTestBase::makeHiveInsertTableHandle(
   VELOX_CHECK_EQ(numPartitionColumns, partitionedBy.size());
   VELOX_CHECK_EQ(numBucketColumns, bucketedBy.size());
   VELOX_CHECK_EQ(numSortingColumns, sortedBy.size());
+
+  std::function<std::shared_ptr<dwio::common::FlushPolicy>()>
+      flushPolicyFactory = nullptr;
+  if (tableStorageFormat == dwio::common::FileFormat::DWRF) {
+    flushPolicyFactory = [&stripSize]() {
+      return std::make_shared<velox::dwrf::DefaultFlushPolicy>(
+          toCapacity(stripSize, core::CapacityUnit::BYTE), 16777216);
```

Comment:
Are we verifying this policy in the test?

Reply:
@majetideepak Yes, by setting this policy, it will supersede the default policy configuration found [here](https://github.com/facebookincubator/velox/blob/main/velox/dwio/dwrf/writer/Writer.cpp#L84).